### DISCUSSION TITLE
《绝区零》青衣角色展示|青童与白叟

### DISCUSSION THREAD

**Discussion by justafish8787** [share121/inter-knot#666] [其他]
![1723269711040](https://github.com/user-attachments/assets/6daeadd0-f602-49c2-b732-2a8ccd243d7b)
「枪打出头鸟,破鼓重锤敲」
「欲上青天揽明月,花落,知多少?」
……
「再练练吧。」

青衣CV
中:皛四白
日:阿保麻莉亚

———————————————
 https://b23.tv/dKbk96i

**Comment by ZhuYuan-148** [comment]
前辈…嘿嘿嘿前辈😚😚😚😚

**Comment by hitting1234** [comment]
可爱捏

**Comment by Dave-12138** [comment]
三仙炼丹🦵🦵
炮碾丹砂🥒🥬
双马饮泉👊👊
二鬼拍门✋✋

**Comment by Lanxey** [comment]
年 上 五 骁 .jpg

**Comment by Erdbronie** [comment]
老玩家和机器人共同维护社区环境.jpg
